src/org/dokuwiki/translatorBundle/Resources/views/Extension/edit.html.twig

```diff
@@ -0,0 +1,60 @@
+{% extends 'dokuwikiTranslatorBundle:Default:index.html.twig' %}
+
+{% block content %}
+    <ul class="breadcrumb">
+        <li><a href="{{ path('dokuwiki_translator_homepage') }}">Home</a> <span class="divider">/</span></li>
+        <li>
+            {% if repository.type == 'core' %}
+                <a href="{{ path("dokuwiki_translator_show") }} ">{{ repository.displayName|capitalize }}</a>
+            {% else %}
+                <a href="{{ path("dokuwiki_translator_show_extension", {'type': repository.type, 'name': repository.name}) }} ">{{ repository.displayName|capitalize }}</a>
+            {% endif %}
+            <span class="divider">/</span>
+        </li>
+        <li class="active">Edit Settings</li>
+    </ul>
+    <div class="container">
+        <div class="row">
+            <div class="span12">
+                <div class="page-header">
+                    <h1>Edit {{ form.vars.data.type|capitalize }} Settings</h1>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="span6">
+                <p>
+                    Here you can edit the settings of your {{ form.vars.data.type }}. You can
+                    use this edit URL one time. After a succesfull save of this form you need to
+                    request a new edit URL for eventually other edits.
```

Comment:
I think eventually is the wrong word here. How about "...for any additional edits"?